app/src/test/java/tttTDD/Java/BoardTest.java

```diff
@@ -8,23 +8,25 @@ public class BoardTest {
     Board board;
 
     @Test
-    public void getBoardInstanceWithCorrectLength() {
-        board = new Board();
+    public void getBoardInstanceWithCorrectDefaultLengthOf3() {
+        board = new Board(3);
```

Comment:
The name of this test implies that you're going to leave length unspecified, but then you do.